pkgs/development/compilers/gerbil/build.nix

```diff
@@ -32,26 +29,34 @@ stdenv.mkDerivation rec {
       substituteInPlace "$f" --replace '"gsc"' '"${gambit}/bin/gsc"'
     done
     substituteInPlace "etc/gerbil.el" --replace '"gxc"' "\"$out/bin/gxc\""
-
-    cat > etc/gerbil_static_libraries.sh <<EOF
-#OPENSSL_LIBCRYPTO=${makeStaticLibraries openssl}/lib/libcrypto.a # MISSING!
-#OPENSSL_LIBSSL=${makeStaticLibraries openssl}/lib/libssl.a # MISSING!
-ZLIB=${makeStaticLibraries zlib}/lib/libz.a
+'';
+
+## TODO: make static compilation work.
+## For that, get all the packages below to somehow expose static libraries,
+## so we can offer users the option to statically link them into Gambit and/or Gerbil.
+## Then add the following to the postPatch script above:
+#     cat > etc/gerbil_static_libraries.sh <<EOF
+# OPENSSL_LIBCRYPTO=${makeStaticLibraries openssl}/lib/libcrypto.a # MISSING!
```

Comment:
I've not used `makeStaticLibraries` before, but the libs are actually in `openssl.out`, so ${makeStaticLibraries openssl.out}/lib/libcrypto.a`, might work here.

Reply:
On my NixOS machine:
```
# ls -ld /nix/store/*openssl*/lib/*.a
/run/current-system/sw/bin/ls: cannot access '/nix/store/*openssl*/lib/*.a': No such file or directory
```
so you do need `makeStaticLibraries`, except all it does is set a configure flag, and there are a lot of libraries with which that doesn't actually work, and some of these libraries would be required to do non-trivial static linking with gerbil and gambit. Sigh.